lib/models/page.js

```diff
@@ -183,6 +183,22 @@ module.exports = function(crowi) {
     })
   }
 
+  // Unlink: Remove redirect origin page
+  pageSchema.methods.unlink = async function(userData) {
+    const Page = crowi.model('Page')
+    if (this.isUnlinkable(userData)) {
+      debug('Unlink page', this._id, this.path)
+      try {
+        const redirectPage = await Page.removePageById(page._id)
```

Comment:
Here 

> ReferenceError: page is not defined

occurred.
It happened when rename the page to the path which exists as the redirect page.